03 Misc/06 Production Configuration/dev.webpack.config.js

```diff
@@ -1,9 +1,49 @@
+var path = require('path');
+var webpack = require('webpack');
 var webpackMerge = require('webpack-merge');
 var commonConfig = require('./base.webpack.config.js');
 
+var basePath = __dirname;
+
 module.exports = function () {
   return webpackMerge(commonConfig, {
     // For development https://webpack.js.org/configuration/devtool/#for-development
     devtool: 'inline-source-map',
+
+    module: {
+      rules: [
+        {
+          test: /\.scss$/,
+          exclude: /node_modules/,
+          loaders: [
```

Comment:
I'm using the new convention [see here](https://github.com/Lemoncode/webpack-2.x-by-sample/pull/76/files#diff-17706b577dd49fa0f440039c4f895e06L39). 

[Rule.loaders](https://webpack.js.org/configuration/module/#rule-loaders) exists only for compatibility reasons.